embedding/mbedk64f/js2c.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
```

Comment:
I accidentaly realised that `js2c.py` is doing same thing as `./tools/generator.sh`. Additionally, `./embedding/mbedk64f/js2c.py` and `./embedding/esp8266/js2c.py` looks nearly the same.